setup_ws.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+echo sourcing /opt/ros/melodic/setup.bash
+source /opt/ros/melodic/setup.bash
+
+echo sourcing catkin_ws/devel/setup.bash
+
+echo exporting variable TURTLEBOT3_MODEL=burger
+export TURTLEBOT3_MODEL=burger
+
+
+
+# tmux
+tmux start-server
+tmux new-session -d -s padawan
+
+
+# RUN roslaunch turtlebot3_gazebo turtlebot3_empty_world.launch
+# Screen 0: gazebo
+tmux rename-window -t padawan:0 gazebo
+tmux send-keys -t padawan:0 "roslaunch turtlebot3_gazebo turtlebot3_empty_world.launch" C-m
+
+# Screen 1: shell
+tmux new-window -t padawan:1
+tmux rename-window -t padawan:1 'shell'
+tmux send-keys -t padawan:1 "source catkin_ws/devel/setup.bash" C-m
+
+
+# attach
+tmux attach
```

Comment:
it would be good to keep the `Dockerfile` and helper scripts in a separated folder

Reply:
done